ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -233,27 +318,58 @@ const LocalStandardDisks = ({ idPrefix, setIsFormValid, onAddErrorNotification }
         }
     ));
 
+    const dropdownBulkSelect = (
+        <DropdownBulkSelect
```

Comment:
I find it somewhat confusing to have two things with same name differing only in first letter upper/lower case.